### DISCUSSION TITLE
CCU3 looses IP and network connectivity

### DISCUSSION THREAD

**Discussion by XinXed** [thkl/hap-homematic#682] [General]
Hello All,

i really love this piece of software which i unfortunately only recently discovered!
everything works extremly well and i can control all my homematic devices - no issue.

but the issue is that from time to time my CCU3 will loose all connectivity - and will also not show in the router client table anymore. 
the programs and scripts from physical buttons still execute fine - so the box is alive.
but i cannot reach it anymore at all by any means (also tried to force getting it a new IP by removing it from the static IP assignment)
This only happens now like every 24 hours roughly after i installed hap-homematic. 
It also seems to happen at random times of the day..

any clues t how i could debug this or even better solve it ? :-D

Thanks!

**Comment by jb-home** [comment]
What kind of network appliance / switch are you using?

Could be that there is some kind of storm control blocking the port your CCU is connected to. That is why you loose connectivity.

Without further information about your infrastructure it is wild guessing...

**Reply by XinXed** [reply]
Thanks for your answer ! 
I use an ASUS AC 5300. Didn’t really come to me that it could be router related. CCU3 is plugged directly into AC5209 via LAN. 
DDOS protections is disabled on the router .. but next time that happens I will just plug the homematic into another network and see what happens. 

the off thing is that I never had that issue before I installed hap. But maybe indeed it’s network spam related as the hap server also throws some error messages for some devices. 
I will attempt to update to RasberryMatic as I read the certain js are outdated on the vanilla CCU that Hap makes us of. Maybe they cause this issue also somehow .

i will continue my debug and trial and error journey :)

**Reply by jb-home** [reply]
HAP-Homematic uses HAP-Bonjour by default. This could be the reason for flooding.
Check the issues here, there is a thread about changing to CIAO as advertisement protocol. That is much better regarding network traffic.

If you have an original CCU3 this might not work for you, as CIAO uses a more recent Node JS version.

**Reply by XinXed** [reply]
Thanks again for your advice ! I will keep that in mind and read into that !
